### PR TITLE
Add SIGs for Docs and Community Engagement

### PR DESCRIPTION
**Pull request by jonasrosland**
We've previously discussed updating the governance doc to include more responsibilities under the "Maintainers" title.

After discussion with @caniszczyk the recommendation is to create something like "Doc Maintainers" or "SIG Docs" or "SIG Community Management" within our MAINTAINERS.md, that way the maintainers description can keep as is and we have more flexibility when we want to add more responsibility to awesome contributors.

This PR adds SIG Docs and SIG Community Engagement, with respective owners for each SIG.

Signed-off-by: Jonas Rosland <jrosland@vmware.com>

### CODE REVIEW

**Comment by xaleeks** [comment]
I like 'SIG Community Management'

**Comment by Vad1mo** [comment]
@jonasrosland I section languages for translation would be a good point as well.

**Review by Vad1mo — APPROVED** [review]

**Review by yanji09 — DISMISSED** [review]

**Comment by OrlinVasilev** [comment]
@goharbor/all-maintainers  can you please review and vote!

**Comment by tianon** [comment]
I think this seems fine, but I wonder about @xaleeks's suggestion -- "SIG Community Engagement" vs "SIG Community Management" doesn't seem very different to me, but maybe Alex can elaborate on why the latter should be preferred? :pray:

**Comment by OrlinVasilev** [comment]
I think it's just wording, in Contour we named it SIG Community and overall that sounds good :)
I aslo like the short form SIG Community - which may include Community + Docs + Site + Events + Webinars etc...

File: MAINTAINERS.md [hunk 1]
```diff
@@ -22,9 +22,18 @@ describes governance guidelines and maintainer responsibilities.
 | Tianon Gravi| [tianon](https://github.com/tianon) | [InfoSiftr](https://github.com/infosiftr) |
 | Vadim Bauer| [Vad1mo](https://github.com/Vad1mo) | [56K.Cloud](https://container-registry.com) |
 
-## Emeritus Maintainers
-* Michael Michael, [michmike](https://github.com/michmike)
-* Nathan Lowe, [nlowe](https://github.com/nlowe)
+## SIG Community Engagement
```
Comment (OrlinVasilev) [inline MAINTAINERS.md:25]: Let's make it "SIG Community" for short and merge it, that took too long :)
```suggestion
## SIG Community
```

**Review by OrlinVasilev — APPROVED** [review]
LGTM lets merge :)

**Comment by OrlinVasilev** [comment]
@tianon @Vad1mo @jonasrosland @steven-zou please review and approve so we can finish this :)

**Review by Vad1mo — APPROVED** [review]

**Review by danielpacak — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
@tianon - are you ok with the proposed changes ? if So please approve :)

**Review by tianon — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
Merged!!! Thank you all!

**Pull request merged**